packages/rancher-sriov/charts/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: sriov
+description: Install SR-IOV operator, cni and device plugin.
+version: 1.0.0
+appVersion: 1.0.0
+home: https://github.com/k8snetworkplumbingwg/sriov-network-operator
+keywords:
+  - sr-iov
+  - Networking
+sources:
+  - https://github.com/rancher/charts
+maintainers:
+  - name: Rancher Labs
+    email: charts@rancher.com
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/experimental: true
```

Comment:
Does this need `catalog.cattle.io/display-name`?

Reply:
According to confluence `  catalog.cattle.io/display-name: "Feature Name" # Only for Charts with custom UI`, as sriov does not have a custom UI, I guess we don't need that